### ISSUE TITLE
Build failing with Flutter 3.13.9

### ISSUE THREAD

**Issue by okmanideep** [JonasWanke/swipeable_page_route#36]
**Describe the bug**
Unable to run the app with the package on Flutter `3.13.7`. This and `3.13.9` are the only versions I tested the package on and it is failing with the following error on `flutter run`
```
../../Pub/Cache/hosted/pub.dev/swipeable_page_route-0.4.1/lib/src/page_route.dart:60:3: Error: The superclass, 'CupertinoPageRoute', has no unnamed constructor that takes no arguments.
  SwipeablePageRoute({
  ^^^^^^^^^^^^^^^^^^
```

**Environment:**
- OS version:  NA
- Flutter version:  3.13.7
- Package version:  0.4.1

**Comment by kvenn** [comment]
I'm also hitting this error. Package seems to not be compatible with latest version of flutter.

**Comment by JonasWanke** [comment]
Sorry for that! I released a new version ([v0.4.2](https://pub.dev/packages/swipeable_page_route/changelog#042--20234-01-20)) that has an updated Flutter version requirement of ^3.16.0. To prevent this from happening again in the future, I configured the CI to also run with the minimum supported Flutter version.
